score/generate_score_config.py

```diff
@@ -105,6 +143,10 @@ def generate_bench_cfg(spec, norm, target):
         help="generated complete benchmark configuration")
     parser.add_argument("--target_score", default=1000,
         help="target score value given these normalizations and specifications")
+    parser.add_argument("--partial",
+                        action='store_true',
+                        help="generates partial config if the benchmark only runs part of the spec")
```

Comment:
i think we should take the opportunity to write a more clear explanation of what this is doing. even i forgot how this works at first.  we need to make the whole score thing more user friendly...

in this case what about updating the 'help' for partial to clarify a bit.

help="generates partial config if the benchmark only runs part of the spec.  normally, the spec is supposed to define the set of benchmarks that's expected to exist, and then the provided json data is expected to provide the norm values to match the spec.  To simplify debugging, and not for normal score runs, we allow a convenience for producing a score configuration that matches whatever json data is provided."